domain/camera/src/main/java/com/google/jetpackcamera/domain/camera/CameraXCameraUseCase.kt

```diff
@@ -415,10 +419,14 @@ constructor(
         return eligibleContentValues
     }
 
+    @SuppressLint("MissingPermission")
```

Comment:
Why do we need the MissingPermission suppress lint here? Is it because the other PR isn't merged yet?

If so, you can rebase this PR on top of the other one.